samwise.js

```diff
@@ -7,23 +7,22 @@
 /* global clearTimeout, setTimeout, document */
```

Comment:
SCRIPT-This logic might be clearer if you used a switch statement instead of multiple if-else statements.